src/main/kotlin/org/wa/rceditor/application/Styles.kt

```diff
@@ -1,14 +1,50 @@
 package org.wa.rceditor.application
 
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIcon
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIconView
+import javafx.geometry.Pos
+import javafx.scene.paint.Color
 import javafx.scene.text.FontWeight
-import tornadofx.Stylesheet
-import tornadofx.box
-import tornadofx.cssclass
-import tornadofx.px
+import tornadofx.*
 
 class Styles : Stylesheet() {
     companion object {
+        val promptColor by cssproperty<Color>("-fx-prompt-text-fill")
+
         val heading by cssclass()
+        val prompt by cssclass()
+        val itemRoot by cssclass()
+        val closeIcon by cssclass()
+        val contentLabel by cssid()
+        val addItemRoot by cssclass()
+        val boldLabel by cssclass()
+
```

Comment:
is there a reason for a newline here separating what's above from menuIcon?

Reply:
@sarabiaj there is no reason, probably I just deleted something from there and left empty line